apps/api/src/coscrad-cli/ingest-media-items.cli-command.ts

```diff
@@ -67,10 +73,36 @@ export class IngestMediaItemsCliCommand extends CliCommandRunner {
 
         const generatedIds = await this.idManager.generateMany(partialPayloads.length);
 
+        const mediaLengthMap = new Map<string, number>();
+
+        for (const { filename, mimeType } of partialPayloads) {
+            const ffprobeResult = await ffprobe(`${directory}/${filename}`);
+
+            const { streams } = ffprobeResult;
+
+            const duration = streams?.[0].duration;
```

Comment:
We also need to get the real MIME Type. Add a test case for when the extension and mime type are mutually inconsistent.

Reply:
Let's add a story for this. It's not as direct as I had hoped.

Reply:
[Here](https://www.pivotaltracker.com/story/show/186713526) is the story.